### PR TITLE
Add CODEOWNERS file.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#####################################################
+#
```
Comment (SergeyKanzhelev) [inline CODEOWNERS]: I updated with more comments. Please feel free to revert

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/master/community-membership.md
+# 
+#
+# Learn about CODEOWNERS file format: 
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+*  @bogdandrutu @carlosalberto @SergeyKanzhelev @yurishkuro
```
Comment (SergeyKanzhelev) [inline CODEOWNERS]: Why not all approvers from https://github.com/open-telemetry/community/blob/master/community-members.md#specifications?

Comment (bogdandrutu) [reply CODEOWNERS]: Added only the maintainers, I will add all the approvers :) Sorry.

Comment (SergeyKanzhelev) [reply CODEOWNERS]: fixed

**Pull request merged**